### PR TITLE
Swaps: Create a new swap

### PR DESCRIPTION
**Pull request by dan437**
## Explanation
As a user, I want to submit multiple swaps in a row without clicking on multiple buttons to reset the flow.

We've also changed the button to `Close` after swapping is completed, which redirects a user to the main page.

## Manual testing steps

Make another swap:
- Make a swap
- On the success page click on `Create a new swap`
- The `Build Quote` page is displayed again

Close button:
- Make a swap
- Click on the `Close` button
- The main page is displayed

## Screenshots
Success page with the `Create a new swap` link and `Close` button:
image

### CODE REVIEW

File: ui/pages/swaps/awaiting-swap/awaiting-swap.js [hunk 1]
```diff
@@ -262,12 +285,8 @@ export default function AwaitingSwap({
             );
           } else if (errorKey) {
             await dispatch(navigateBackToBuildQuote(history));
-          } else if (
-            isSwapsDefaultTokenSymbol(destinationTokenInfo?.symbol, chainId)
-          ) {
-            history.push(DEFAULT_ROUTE);
           } else {
-            history.push(`${ASSET_ROUTE}/${destinationTokenInfo?.address}`);
```
Comment (danjm) [inline ui/pages/swaps/awaiting-swap/awaiting-swap.js:270]: Why is this line deleted?

File: ui/pages/swaps/awaiting-swap/awaiting-swap.js [hunk 2]
```diff
@@ -230,6 +236,22 @@ export default function AwaitingSwap({
     );
   }
 
+  const MakeAnotherSwap = () => {
+    return (
+      <Box marginBottom={3}>
+        <a
```
Comment (darkwing) [inline ui/pages/swaps/awaiting-swap/awaiting-swap.js:242]: I think it would be best to use a `<button>` here

**Review by danjm — CHANGES_REQUESTED** [review]
Code looks good, but I left a question about a functionality change.

File: ui/pages/swaps/awaiting-swap/awaiting-swap.js (hunk 2 above)
Comment (dan437) [reply ui/pages/swaps/awaiting-swap/awaiting-swap.js:237]: Thanks for the suggestion! We had a quick chat about when to use a button or a link and since we do page redirection on click here, a link seems fine.

File: ui/pages/swaps/awaiting-swap/awaiting-swap.js (hunk 1 above)
Comment (dan437) [reply ui/pages/swaps/awaiting-swap/awaiting-swap.js:265]: Great question! We synced up yesterday and here is a summary: the requirement is to show the `Close` button after a swap is done and that will redirect to the main page. However, before the swap is done, a user can click on `View in activity`: 
image

Which redirects to the Activity tab, where a user can see progress of their transaction.

Comment (dan437) [reply ui/pages/swaps/awaiting-swap/awaiting-swap.js:265]: An update.. I was redirecting to the Activity tab on the main page. I changed it back to the Activity tab on an asset's page, so it looks like this now: 
image

**Review by danjm — APPROVED** [review]
LGTM!
